cdk/test/cdk.test.ts

```diff
@@ -1,22 +1,12 @@
-import {
-  expect as expectCDK,
-  matchTemplate,
-  MatchStyle,
-} from "@aws-cdk/assert";
-import * as cdk from "@aws-cdk/core";
-import * as Cdk from "../lib/cdk-stack";
+import { SynthUtils } from "@aws-cdk/assert";
+import { App } from "@aws-cdk/core";
+import { CdkStack } from "../lib/cdk-stack";
 
-xtest("Empty Stack", () => {
-  const app = new cdk.App();
-  // WHEN
-  const stack = new Cdk.CdkStack(app, "MyTestStack");
-  // THEN
-  expectCDK(stack).to(
-    matchTemplate(
-      {
-        Resources: {},
-      },
-      MatchStyle.EXACT
-    )
-  );
+describe("The tag-janitor stack", () => {
+  it("matches the snapshot", () => {
+    const app = new App();
+    const role = new CdkStack(app, "tag-janitor");
```

Comment:
Really picky but maybe this should be something other than role...

Reply:
Totally right! Sorry, missed this when cmd+c'ing